ambry-rest/src/main/java/com/github/ambry/rest/ServerSecurityChecker.java

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright 2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.rest;
+
+import com.github.ambry.commons.ServerMetrics;
+import com.github.ambry.server.ServerSecurityService;
+import io.netty.channel.ChannelHandler;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.channel.ChannelInboundHandlerAdapter;
+import io.netty.handler.ssl.SslHandler;
+import java.util.Objects;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * Handler that performs any security validation on the new HTTP2 connection.
+ */
+@ChannelHandler.Sharable
+public class ServerSecurityChecker extends ChannelInboundHandlerAdapter {
```

Comment:
serverSecurityChecker -> serverSecurityHanlder ? To make it netty style.

Reply:
rename the class as suggested. serverSecurityChecker -> serverSecurityHandler.